core/dbt/config/project.py

```diff
@@ -59,6 +59,11 @@
 --no-version-check
 """
 
+MALFORMED_PACKAGE_ERROR = """\
+packages.yml is malformed, please make sure that it matches 
```

Comment:
Check out these copy suggestions. Couple of things:
1. We like using full sentences in these error messages
2. I picked a more generic link so we don't accidentally break this in the docs in the future
3. We should retain the error message (see below) in this error output

```suggestion
The packages.yml file in this project is malformed. Please double check the contents
of this file and fix any errors before retrying.

You can find more information on the syntax for this file here:
https://docs.getdbt.com/docs/package-management

Validator Error:
{error}
```

Reply:
noted, though regarding validation error retainment - the initial issue was raised that it is misleading, so i was doubtful regarding including it into the message